migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_030__vedtaksbrevvalg_fritekstbrev.sql

```diff
@@ -0,0 +1,27 @@
+create table if not exists vedtaksbrev_valg
+(
+    id                 bigint                                not null primary key,
+    behandling_id      bigint                                not null references behandling (id),
+    redigert           boolean                               not null,
+    redigert_brev_html text,
+    hindret            boolean                               not null,
+    aktiv              boolean     default true,
+    versjon            bigint      default 0                 not null,
+    opprettet_av       varchar(20) default 'VL'              not null,
+    opprettet_tid      timestamp   default CURRENT_TIMESTAMP not null,
+    endret_av          varchar(20),
+    endret_tid         timestamp,
+    constraint fk_behandling foreign key (behandling_id) references behandling (id)
+);
+
+create unique index if not exists idx_vedtaksbrev_valg_behandling_aktiv
+    on vedtaksbrev_valg (behandling_id)
+    where aktiv = true;
+
+create sequence if not exists seq_vedtaksbrev_valg increment by 50 minvalue 1000000;
+comment on column vedtaksbrev_valg.redigert is 'Indikerer om sakbehandler har redigert eller skrevet et fritekstbrev';
+comment on column vedtaksbrev_valg.redigert_brev_html is 'Html tekst som skal brukes istedenfor det automatiske brevet ';
+comment on column vedtaksbrev_valg.hindret is 'Indikerer om brevet er manuelt hindret';
+comment on column vedtaksbrev_valg.aktiv is 'Soft delete';
```

Comment:
Litt rar kommentar på aktiv-flagg, men la gå.

Reply:
Kunne kanskje ha utvidet til "Flagg som sier om brevvalget er aktivt for å fasilitere soft delete"

Reply:
jeg fjernet feltet. Tar det heller inn hvis det blir relevant.